vm_exec.h

```diff
@@ -64,6 +64,7 @@ error !
 
 #define START_OF_ORIGINAL_INSN(x) /* ignore */
 #define DISPATCH_ORIGINAL_INSN(x) return LABEL(x)(ec, reg_cfp);
+#define CURRENT_INSN_IS(pop) ((rb_insn_func_t)GET_CURRENT_INSN() == LABEL(pop))
```

Comment:
ok to use `pop`? (`insn`?)

Reply:
It's OK because currently pop is the only usage.  But insn may be a polite name.